src/Core/Components/Dialog/FluentDialog.razor.css

```diff
@@ -1,14 +1,13 @@
 ﻿fluent-dialog::part(control) {
-    --dialog-padding: 8px;
+    --dialog-padding: calc(var(--design-unit) * 6 * 1px);
     position: relative;
     padding: 0px var(--dialog-padding);
     width: calc(var(--dialog-width) - 2 * var(--dialog-padding));
     height: calc(var(--dialog-height) - 2 * var(--dialog-padding));
-    max-width: 90%;
```

Comment:
I think it's best to have a maximum size for the dialog, otherwise we could have a "full dialog" like this one.

![image](https://github.com/microsoft/fluentui-blazor/assets/8350694/547d01f3-af2f-4a87-b355-e080475a351a)

Reply:
At 90% it is still a "full dialog". 80% fixes it
<img width="259" alt="image" src="https://github.com/microsoft/fluentui-blazor/assets/1761079/4774c4fe-5838-4a95-8b14-6a207cd2140d">